ArtOfIllusion/src/artofillusion/ObjectTextureDialog.java

```diff
@@ -145,15 +144,7 @@ public ObjectTextureDialog(LayoutWindow parent, ObjectInfo objects[], boolean in
     
     for (Texture texture : PluginRegistry.getPlugins(Texture.class))
     {
-      try
```

Comment:
If we eliminate the use of reflection, then they would not be necessary.  I'll have to take a little more time to consider the other comments on this PR.